src/components/side-panel/layer-panel/layer-configurator.js

```diff
@@ -529,8 +525,12 @@ export default class LayerConfigurator extends Component {
 
     return (
       <StyledLayerVisualConfigurator>
-        {/* Color By */}
-        <LayerConfigGroup label={'color'} collapsible>
+        {/* Fill Color */}
+        {featureTypes.polygon || featureTypes.point? (<LayerConfigGroup
```

Comment:
if you want to avoid the ternary operator __?__ and the __: null__, you can do simply
```
(featureTypes.polygon || featureTypes.point) && (...)
```

Reply:
I think `? : null` has been a pattern in all our components, so I prefer to keep it this way. Also if it's in Component render function `(Boolean(a) && <div/>)` will result in `false` been rendered on the screen when Boolean(a) is falsy